### ISSUE TITLE
Camera doesn't show

### ISSUE THREAD

**Issue by krajzler** [Dhaval2404/ImagePicker#143]
If I want to select image from camera, nothing happens. I tried on two different ways:

```
ImagePicker.with(this)
        .crop()
        .start()
```
Nothing happens if I click on Camera button. And:
```
ImagePicker.with(this)
        .crop()
        .cameraOnly()
        .start()
```
Nothing happens at all. There is also nothing in logs.

I'm using Google Pixel 3 Android 11 (API 30).

**Comment by Dhaval2404** [comment]
@krajzler Thank you so much for reporting the issue.

Today, I've tried to reproduce the issue using the given code but it seems to be working fine on my end. Here is the video for reference: [Sample Emulator Video](https://user-images.githubusercontent.com/7314181/103093592-cbfa2c00-4620-11eb-9f50-914868d005d3.mp4)  and  [Sample Pixel 4a Video](https://user-images.githubusercontent.com/7314181/103093842-93a71d80-4621-11eb-9e2b-63e7a459ad48.mp4)

Gradle: 
```
implementation 'com.github.dhaval2404:imagepicker:1.8'
```

Sample Code:

```Kotlin
ImagePicker.with(this)
            .crop()
            .cameraOnly()
            .start()
```

Device Tested: **Pixel 2 API 30** (Emulator) and **Pixel 4a API 30** (Real Device)

It would be a great help to me if you can provide any steps with which I can reproduce the issue. Looking forward to your reply.

**Comment by odolejsi** [comment]
I just ran into this as well after updating to 1.8 (to fix the crash with uCrop on super high resolution phones).
The error coming from ImagePicker is:
```java
java.lang.Throwable: Camera app not found
```

**Comment by nothingimportant55** [comment]
Same issue here. Pixel 2, API 30. Camera doesn't appear

**Comment by Dhaval2404** [comment]
@krajzler @nothingimportant55 

I found that the Checking Camera App Existing or not is not working in an app that targets API 30 (targetSdkVersion 30). It is due to the [package-visibility](https://developer.android.com/training/basics/intents/package-visibility)  introduced in API 30. I'm working on a possible solution. Hopefully, I will release the updated version by EOD.

Thank you so much for your feedback,

**Comment by Dhaval2404** [comment]
For an immediate solution, please add **queries** into your AndroidManifest.xml

```xml
<manifest package="com.github.dhaval2404.imagepicker">

    <application>


    </application>


    <queries>
        <!-- Camera -->
        <intent>
            <action android:name="android.media.action.IMAGE_CAPTURE" />
        </intent>
    </queries>

</manifest>

**Comment by sagarkokare** [comment]
If I want to select image from camera, nothing happens.gallery working fine

**Comment by Dhaval2404** [comment]
This issue is resolved in **v2.0**

**Comment by VosoUser** [comment]
i want front camera open. in camera choose camera method needed
